ui/lib/apps/ClusterInfo/components/InstanceTable.tsx

```diff
@@ -153,21 +154,18 @@ export default function ListPage() {
       name: t('cluster_info.list.instance_table.columns.node'),
       key: 'node',
       minWidth: 100,
-      maxWidth: 200,
+      maxWidth: 160,
       isResizable: true,
       columnActionsMode: ColumnActionsMode.disabled,
-      onRender: (node) => (
-        <Tooltip title={`${node.ip}:${node.port}`}>
-          <span>
-            {node.ip}:{node.port}
-          </span>
-        </Tooltip>
-      ),
+      onRender: ({ ip, port }) => {
+        const fullName = `${ip}:${port}`
+        return <Tooltip title={fullName}>{fullName as any}</Tooltip>
```

Comment:
`<Tooltip>` needs some element node I believe, otherwise it won't take effect.

Reply:
But actually it works, the tooltip can be displayed normally, just has type warning.

I just have a look at the Tooltip source code in https://github.com/ant-design/ant-design/blob/master/components/tooltip/index.tsx#L245 :

```js
    const child = getDisabledCompatibleChildren(
      React.isValidElement(children) ? children : <span>{children}</span>,
      prefixCls,
    );
```

So the Tooltip will wrap the string with `<span>`.

Anyway, it is better to pass the element node as children, I will update it soon.

Reply:
Good to know! I didn't know it before.